src/services/feed/register/strategies/twitter.js

```diff
@@ -26,36 +24,15 @@ async function register(data) {
     feed.network_id = feedMeta.account_id;
     feed.meta = JSON.stringify(feedMeta);
 
-    // wait till storage is registered then syncs tweets
     saveAccountJobs.push(storage.feeds().save(feed));
-    syncAccountJobs.push(() => twitter.syncAccount({
-      ...feedMeta,
-      cursor: accountsCursor[feed.network_id],
-    }));
   }
 
   await Promise.all(saveAccountJobs);
+  // saved accounts will be synced upon next tweet sync cycle
 
   // return amount of accounts
   logger.info(`Saved ${accounts.length} accounts`);
 
-  (async () => {
-    try {
-      const results = await Promise.allSettled(syncAccountJobs.map((job) => job()));
-
-      for (const [idx, result] of results.entries()) {
-        if (result.status !== 'fulfilled') {
-          logger.warn({ err: result.reason, account: accounts[idx] }, 'failed to sync');
-        }
-      }
-
-      // update twitter feed
-      twitter.connect();
-    } catch (e) {
-      logger.error({ err: e }, 'failed to perform async op');
-    }
-  })();
-
```

Comment:
needs explanation why it is removed, why would it not be under a flag? ie we can put it into config and cover it with enable/disable

Reply:
Here the logic was 
1. save new account into database 
2. immediately do sync job on the account
It required official twitter client and unlimited access to api. But there is no official twitter client now.  Also connect method is no-op right now 

When using nitter the synchronization cycle is implemented within single timer. It started upon plugin initialization and controls all the sync work for all twitter accounts in single loop. Since number of simultaneous calls within twitter guest account is limited, it is important not to start multiple simultaneous fetch requests, otherwise we risk to encounter "too many requests error". 
for nitter the logic is
1. save new account into database
2. wait for the next sync cycle
3. load accounts from db and fetch page by page from newest to oldest all new tweets for each of them until we found the tweet that already exists in the db...